test/integration/test/philips_light_test.js

```diff
@@ -38,57 +35,117 @@ Prepper.makeSuite('Control lights locally',function(){
   });
 
   describe ('once logged in', function () {
+    var lightKeys;
+
     before(function() {
+      // turn off all lights in the simulator
+      Prepper.philipshue_server.turnOffLight(1);
+      Prepper.philipshue_server.turnOffLight(2);
+      Prepper.philipshue_server.turnOffLight(3);
+
       return chakram.post(loginURL,null,{'headers' : header})
       .then(function(resp){
        header.Authorization = 'Bearer ' + resp.body.session_token;
        chakram.setRequestDefaults({'headers': header});
+
+       expect(Prepper.philipshue_server.lightStatus(1) &&
+        Prepper.philipshue_server.lightStatus(2) &&
+        Prepper.philipshue_server.lightStatus(3)).equals(false);
      });
     });
 
     it('check 3 bulbs are registered',function(){
 
-      return chakram.get(serviceListURL)
+      // collect all getters for the lightbulbs
+      var getURL = channelURL + '/get';
+      var getterPayload = [{'kind': 'LightOn'}];
+      
+      return chakram.put(getURL,getterPayload)
```

Comment:
I think we could add more context by putting this call in `philipshue_server.js`. We could name it `fetchAllGetters`. What do you think?